cgi-bin/LJ/Global/Defaults.pm

```diff
@@ -260,14 +260,7 @@ no strict "vars";
     # 'type' is used as 'type' attribute value in 'codes' table
     # 'save_region_code' specifies what to save in 'state' userprop  -
     # '1' mean save short region code and '0' - save full region name
-    %LJ::COUNTRIES_WITH_REGIONS = (
```

Comment:
I killed RU here because... we don't do it for Canada, so whatever. (And inserting the plain RU states was causing the strict mode SQL to be angry since it's not latin1.)